packages/amplify-category-hosting/__tests__/lib/S3AndCloudFront/configuration-manager.test.js

```diff
@@ -0,0 +1,108 @@
+jest.mock('inquirer');
+const inquirer = require('inquirer');
+
+jest.mock('../../../lib/S3AndCloudFront/helpers/configure-Website');
```

Comment:
Nit: I usually prefer to group all my `require` at the top of the file and group all `jest.mock` together. It makes it easy to read and figure out if something is mocked or not